File: C Programming/problem on numbericals/problem6.c
/*
   6.Write a program which accept one number from user and print that number of even numbers on screen. 
	
	Author: Milind Nalawade
	
	Date: 19/06/2021

	language: C language
*/

#include<stdio.h>

//Entry point function

int main()
{
	int a=0,i=2;
	printf("Enter your number : ");
	scanf("%d",&a);
	while(a)
	{
		printf("%d ",i);
		i=i+2;
		a--;
	}
	return 0;
}

File: Python Programming/problems on numbericals/problem6.py
'''
6.Write a program which accept one number from user and print that number of even numbers on screen. 
	
	Author: Milind Nalawade
	
	Date: 19/06/2021

	language: Python 3.9.0
	
'''
#==============
#Entry point function
#==============

def main():
	no=int (input("Enter your number : "))
	i=2;
	while no!=0:
		print(i,end=" ")
		i=i+2
		no=no-1
		
#==========
#starter
#==========
if __name__=="__main__":
	main()
